README.md

```diff
@@ -47,6 +46,7 @@ brew install socket_vmnet
 ```
 
 The binaries will be installed onto the following paths:
+
```

Comment:
Is this spacing required by some linter? If so, it should be enforced in the CI

Reply:
BTW what is the benefit of this spacing style?

Reply:
We can use yamllint to enforce this. I'll add an issue to add it.

Reply:
The source is more clear. The rendered page looks the same.